### PR TITLE
Cleanup 

### PR DESCRIPTION
**Pull request by sktatsuno**
Added comments, removed redundant comments, renamed variables/functions for clarity

### CODE REVIEW

File: src/denostore.ts [hunk 1]
```diff
@@ -60,37 +60,36 @@ export default class Denostore {
...
       this.#schema = schema;
     }
   }
 
+  /** 
+   ** Caching method to be invoked in the field resolver of queries user wants cached
+   ** Creates cache key by accessing resolver 'info' AST for query information
+   ** Accepts optional expire time in seconds argument
+   ** Retrieves cached value using created cache key
+   ** If cache key does not exist for a query, invokes provide callback and sets cache with results
```
Comment (jakedoublev) [inline src/denostore.ts:79]: typo - "invoked provide_d_ callback" missing the d

File: src/denostore.ts [hunk 2]
```diff
@@ -101,42 +100,47 @@ export default class Denostore {
...
     if (opts) {
       await this.#redisClient.set(
-        queryExtractName,
+        cacheKey,
         JSON.stringify(results),
         opts
       );
-      // if negative expire argument specified or if no options specified set cache with no expiration
+      /**
+       * If negative expire argument provide or no expire specified, cache results with no expiration
```
Comment (jakedoublev) [inline src/denostore.ts:122]: If negative expire argument "provided" missing the d

File: src/denostore.ts [hunk 3]
```diff
@@ -158,24 +162,26 @@ export default class Denostore {
...
+          // pass denostore instance through context to use methods in resolvers
           contextValue: { denostore: this },
           variableValues: variables,
         });
+
         // if errors delete results data
-        results.errors ? delete results.data : null;
-        response.status = results.errors ? 500 : 200;
-        response.body = results;
+        graphqlResults.errors ? delete graphqlResults.data : null;
```
Comment (jakedoublev) [inline src/denostore.ts:181]: should this just be an "if" statement instead of ternary?

File: src/utils.ts [hunk 4]
```diff
@@ -1,22 +1,22 @@
...
     console.error(
       '%cDenostore cache function does not allow caching of mutations.',
       'font-weight: bold; color: white; background-color: red;'
     );
     throw new Error('Query error. See server console.');
   }
-  const node = info.fieldNodes[0];
-  const queryObj: QueryObjType = {
+  const node: FieldNode = info.fieldNodes[0];
+  const cacheKeyObj: CacheKeyObj = {
```
Comment (jakedoublev) [inline src/utils.ts:13]: should the type "CacheKeyObj" be called "CacheKeyObjType"?

**Review by jakedoublev — CHANGES_REQUESTED** [review]
Left a few comments that could use review. Great work, Scott.

File: src/denostore.ts (hunk 3 above)
Comment (sktatsuno) [reply src/denostore.ts:181]: @Jamesmjkim was there a specific reason you used a ternary here? Was it to match the line below it?

File: src/utils.ts (hunk 4 above)
Comment (sktatsuno) [reply src/utils.ts:13]: Thanks for the catch I actually need to change "QueryObjType" to just "QueryObj"

File: src/denostore.ts (hunk 3 above)
Comment (Jamesmjkim) [reply src/denostore.ts:181]: @sktatsuno No particular reason. I just thought ternary looked clean here. You can just use "if" statement if you want.

File: src/utils.ts (hunk 4 above)
Comment (sktatsuno) [reply src/utils.ts:13]: No I don't think the "Type" is necessary because our other defined types don't have "Type" at the end

**Review by Jamesmjkim — COMMENTED** [review]
Looks good to me. Great job Scott!

**Review by Jamesmjkim — APPROVED** [review]

**Pull request merged**